src/test/java/org/mockitousage/verification/VerificationWithTimeoutTest.java

```diff
@@ -121,7 +121,7 @@ public void shouldAllowMixingOnlyWithTimeout() throws Exception {
         verify(mock, timeout(40).only()).clear();
     }
 
-    @Test
+    @Test(expected=NoInteractionsWanted.class)
```

Comment:
This test was flickering, so I extended its timeout (40 -> 50). In so doing, I had to understand what the test was up to, so I added comments to aid future readers and took out the counter intuitive exception expectation, replacing it with the default JUnit method.